### PR TITLE
OKTA-505386 Revise Authentication and End User Rate Limits reference

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updates to this guide: https://developer.okta.com/docs/reference/rl-global-enduser/ Removed some note formatting; added a line to use the Rate Limit dashboard limits not in list; revised Identity Engine description to define goal of "support" of 1000 authentication flows. 
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-505386](https://oktainc.atlassian.net/browse/OKTA-505386)

### CODE REVIEW

**Review by joec-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 1]
```diff
@@ -7,18 +7,14 @@ excerpt: >-
 
 # Authentication/End-user rate limits
 
-This page provides the API rate limits for authentication and end-user activities, which is part of Okta [rate limits](/docs/reference/rate-limits).
-
-> **Note:**
->
-> * To learn more about rate limits, visit our [overview](/docs/reference/rate-limits) and [best practices](/docs/reference/rl-best-practices) pages.
+This page provides the API rate limits for authentication and end-user activities, which is part of Okta [rate limits](/docs/reference/rate-limits). To learn more about rate limits, visit our [overview](/docs/reference/rate-limits) and [best practices](/docs/reference/rl-best-practices) pages.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:10]:
```suggestion
This page provides the API rate limits for authentication and end-user activities, which is part of Okta [rate limits](/docs/reference/rate-limits). To learn more about rate limits, visit our [Overview](/docs/reference/rate-limits) and [Best practices](/docs/reference/rl-best-practices) pages.
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 2]
```diff
@@ -7,18 +7,14 @@ excerpt: >-
...
 
 > * In addition to the rate limit per API, Okta implements limits on concurrent requests, Okta-generated email messages, end user requests, and home page endpoints. These limits are described on the [Additional limits](/docs/reference/rl-additional-limits/) page.
 > * [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/) apply to a variety of endpoints across different APIs for customers that purchased this add-on.
 > * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
 
-> We enforce the following **per-minute** limits.
+See the following list of **per-minute** limits. If an endpoint is not in this list, you can review it using the Admin Console, in the Rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:17]:
```suggestion
See the following list of **per-minute** limits. If an endpoint is not in this list, you can review it using the Admin Console, in the rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
```

**Review by susanharper-okta — APPROVED** [review]
couple baby nits approving to not hold you up

**Pull request merged**